<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string/>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>191</width>
           <height>79</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="1">
           <widget class="QRadioButton" name="freeze">
            <property name="text">
             <string>Freeze</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="powerOn">
              <property name="text">
               <string>Power ON</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="powerOff">
              <property name="text">
               <string>Power OFF</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Способ Подключения</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="1">
          <widget class="QPushButton" name="Confirm">
           <property name="text">
            <string>Confirm</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="com">
           <property name="title">
            <string>Com Порт</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <widget class="QLineEdit" name="comName">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>30</y>
              <width>113</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>COM1</string>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>81</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Имя COM порта</string>
            </property>
           </widget>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="eth">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>Ethernet</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <widget class="QLineEdit" name="ethName">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>50</y>
              <width>113</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>8080</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>50</y>
              <width>81</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Имя Порта</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="ethName_2">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>20</y>
              <width>113</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>localhost</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>81</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Адрес</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string/>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>322</width>
           <height>85</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QPushButton" name="menuKey">
            <property name="text">
             <string>Menu Key</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" rowspan="2">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0" rowspan="2">
             <widget class="QPushButton" name="leftKey">
              <property name="text">
               <string>←</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="upKey">
              <property name="text">
               <string>↑</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2" rowspan="2">
             <widget class="QPushButton" name="rightKey">
              <property name="text">
               <string>→</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="downKey">
              <property name="text">
               <string>↓</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="enterKey">
            <property name="text">
             <string>Enter Key</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="defaultKey">
            <property name="text">
             <string>Default Key</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="4" column="1" colspan="2">
          <widget class="QComboBox" name="testPattern">
           <item>
            <property name="text">
             <string>OFF</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>All white</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>All black</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1% Window</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1% Window (Inverion)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Focus</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Color bar (vertical)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Color bar (horizontal)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16:9/4:3 aspect frame</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Input Select</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Numeric Key</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Test Pattern</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="5">
          <widget class="QSlider" name="horizontalSlider">
           <property name="maximum">
            <number>9</number>
           </property>
           <property name="pageStep">
            <number>3</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="5">
          <widget class="QPushButton" name="sysSelector">
           <property name="text">
            <string>System Selector</string>
           </property>
          </widget>
         </item>
         <item row="2" column="6">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="2">
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PC1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PC2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RGB1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>RGB2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Video</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DVI</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>HDMI</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Digital Link</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string/>
           </property>
           <widget class="QRadioButton" name="radioButton">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>82</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>On Screen</string>
            </property>
           </widget>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QPushButton" name="func3Key">
           <property name="text">
            <string>Function 3 Key</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="func2Key">
           <property name="text">
            <string>Function 2 Key</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="func1Key">
           <property name="text">
            <string>Function 1 Key</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string/>
           </property>
           <widget class="QRadioButton" name="AVMute">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>63</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>AV Mute</string>
            </property>
           </widget>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="autoSetup">
           <property name="text">
            <string>Auto Setup</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
